src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,21 +4,38 @@
 
 package frc.robot;
 
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.XboxController.Button;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import frc.robot.Constants.OIConstants;
+import frc.robot.subsystems.SpeakerTop;
 
 public class RobotContainer {
+  private final SpeakerTop m_speakerShooter = new SpeakerTop();
+  private XboxController m_driverController = new XboxController(OIConstants.kDriverControllerPort);
 
   // Instance variables go here. This typically includes all robot systems and controllers.
 
   public RobotContainer() {
-    // Initialization goes here...
+    m_speakerShooter.setDefaultCommand(
```

Comment:
Instead of using `setDefaultCommand`, you may find it easier to use the `finallyDo` decorator on the `new RunCommand`s below.